src/GrpcProtos/capture.proto

```diff
@@ -298,6 +314,7 @@ message ClientCaptureEvent {
     // it is going to go away in the future when we switch to
     // frame-pointer based unwinding.
     AddressInfo address_info = 16;
+    ApiEvent api_event = 24;
```

Comment:
As per the comment at the beginning of the `oneof`, this should probably be in the 1-15 range as it's high frequency. Unless we already know that the field number will likely change, in which case we don't want to take away a field number in the 1-15 range.

Reply:
cool, I made the change, thanks.